documentation/user-consent.md

```diff
@@ -12,7 +12,7 @@ This document captures guidelines for Operator platform to handle user consents.
     - [Purpose levels](#purpose-levels)
     - [Purpose relationship with other key concepts](#purpose-relationship-with-other-key-concepts)
     - [Purpose definition](#purpose-definition)
-    - [Using purpose parameter in the authorization request](#using-purpose-parameter-in-the-authorization-request)
+    - [Applying purpose concenpt in the authorization request](#applying-purpose-concenpt-in-the-authorization-request)
```

Comment:
"concenpt" --> "concept" in 2 places

Reply:
Thanks. Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/13/commits/5ab70dc278f926455ece38c1294586ed1c84dfa6